pcdsdevices/device_types.py

```diff
@@ -42,7 +42,7 @@
 from .sensors import RTD, TwinCATThermocouple
 from .sequencer import EventSequencer
 from .slits import Slits
-from .spectrometer import Kmono, VonHamos4Crystal
+from .spectrometer import Kmono, VonHamos4Crystal, OldVonHamos4Crystal
```

Comment:
```suggestion
from .spectrometer import Kmono, OldVonHamos4Crystal, VonHamos4Crystal
```

The pre-commit checks failed here, asking for these to be listed in alphabetical order